modules/ols-0-2-1-release-notes.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+// release_notes/ols-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ols-0-2-1-release-notes_{context}"]
+= {ols-long} version 0.2.1
+
+{ols-official} 0.2.1 is now available on {ocp-product-title} 4.15 or later.
+
+[id="ols-0-2-1-enhancements_{context}"]
+== Enhancements
+
+The following enhancements have been made with {ols-official} 0.2.1:
+
+* With this release, the {ols-long} User Interface is hidden if the *Hide Lightspeed* option is selected in *User Preferences*.
+** Xavier to review in latest product build
```

Comment:
@kyoto I am not seeing this option in *User Preferences*. when I installed v0.2.1 Could you please check